src/metamodels/crosscat.py

```diff
@@ -228,15 +228,31 @@ class CrosscatMetamodel(metamodel.IBayesDBMetamodel):
     with names that begin with ``bayesdb_crosscat_``.
 
     `crosscat` can be a crosscat engine instance or constructor. Prefer to pass
-    a constructor, since that way the engine obtains a random seed from the
-    BayesDB object. The 'crosscat as instance' case is kept for backwards
-    compatibility.
+    a constructor, since that way a new crosscat engine is constructed for each
+    query, and its random seed is taken from the BayesDB object. The 'crosscat
+    as instance' case is kept for backwards compatibility.
 
     Arguments to the constructor can be passed by ccargs and cckwargs. If using
     a crosscat MultiprocessingEngine, the process pool must be passed as a
     'pool' argument via cckwargs. It would not do for a new process pool to be
     created every time a new query is made.
 
+    E.g.
+
+    >>> import bayeslite.metamodels.crosscat as cc
+    >>> from crosscat import MultiprocessingEngine as mpe
+    >>> import bayeslite.metamodels.crosscat as cc
+    >>> from bayeslite import bayesdb_register_metamodel as rg
+    >>> from bayeslite import bayesdb_open as bp
+    >>> pool = mpe.Pool(4)
+    >>> thelambda = lambda: mpe.MultiprocessingEngine(pool=pool)
+    >>> rg(bp(builtin_metamodels=False), cc.CrosscatMetamodel(thelambda))
```

Comment:
These names are likely to be copied into client code. Would recommend slightly longer and more meaningful names, even if the registration command takes multiple lines.